src/conv/k-g/region_list.cpp

```diff
@@ -49,16 +49,53 @@ Bot& RegionList::addRegion
 }
 
 
+void RegionList::setAttributes
+(
+    const std::string&                        name,
+    const std::map<std::string, std::string>& attributes
+) {
+    m_list[name].attributes = attributes;
+}
+
+static void  writeAttributes
+(
+    rt_wdb* wdbp,
+    const char* name,
+    const std::map<std::string, std::string>& attributes
+) {
+
+    struct rt_db_internal     region_internal;
+    struct directory*         dp;
+    struct db_i*              dbip;
+    bu_attribute_value_set*   avs;

```

Comment:
This is very much C-style. In C++, you can declare the variables where you need them: declaration = initialization

Reply:
you are right. I will modify it